src/Nethermind/Nethermind.Store/StateDb.cs

```diff
@@ -52,7 +52,25 @@ public StateDb(IDb db)
             set => Set(key, value);
         }
 
-        public byte[][] GetAll() => _db.GetAll();
+        public KeyValuePair<byte[], byte[]>[] this[byte[][] keys]
```

Comment:
extension method, or default interface method

Reply:
This is really based on db implementation